src/core_modules/capture-core/components/WidgetEnrollmentEventNew/Validated/useBuildNewEventPayload.ts

```diff
@@ -25,16 +24,16 @@ export const createServerData = ({
     relationship,
     enrollment,
 }: {
-    serverRequestEvent: RequestEvent,
-    linkedEvent: ?LinkedRequestEvent,
-    relationship: ?Object,
-    enrollment: ?Object,
+    serverRequestEvent: RequestEvent;
+    linkedEvent?: LinkedRequestEvent;
+    relationship?: Record<string, unknown>;
+    enrollment?: Record<string, unknown>;
 }) => {
     const relationships = relationship ? [relationship] : undefined;
     const newEvents = linkedEvent ? [serverRequestEvent, linkedEvent] : [serverRequestEvent];
 
     if (enrollment) {
-        const updatedEnrollment = { ...enrollment, events: [...(enrollment.events || []), ...newEvents] };
+        const updatedEnrollment = { ...enrollment, events: [...((enrollment as any).events || []), ...newEvents] };
```

Comment:
Rollback to `const updatedEnrollment = { ...enrollment, events: [...(enrollment.events || []), ...newEvents] };` and fix the error by using `any` on line 30